clients/drcachesim/tracer/raw2trace.h

```diff
@@ -85,7 +86,10 @@
 #define WINDOW_SUBDIR_FORMAT "window.%04zd" /* ptr_int_t is the window number type. */
 #define WINDOW_SUBDIR_FIRST "window.0000"
 #define TRACE_SUBDIR "trace"
-#ifdef HAS_ZLIB
+
+#ifdef HAS_LZ4
+#    define TRACE_SUFFIX "trace.lz4"
```

Comment:
This should not be the default as lz4 does not support fast skipping: see the other comment.